plutus-use-cases/src/Plutus/Contracts/Uniswap/Types.hs

```diff
@@ -52,9 +52,6 @@ data Liquidity = Liquidity
 PlutusTx.makeIsDataIndexed ''Liquidity [('Liquidity, 0)]
 PlutusTx.makeLift ''Liquidity
 
--- Note: An orphan instance here because of the newtype wrapper below.
-deriving anyclass instance ToSchema AssetClass
```

Comment:
🎉 

i think you can then remove the ghc warning at the top of this file too (no-warn-orphans)

Reply:
Yep, thanks!